### ISSUE TITLE
Nuxt.js  1.0.0 new hook system 

### ISSUE THREAD

**Issue by luchun** [nuxt-community/sentry-module#7]
nuxt.plugin('renderer',..) is deprecated. Please use new hooks system.

please see https://github.com/nuxt/nuxt.js/releases Breaking changes section

**Comment by DiederikvandenB** [comment]
Thanks for reporting this. I will take a look at updating the package either tomorrow or at the start of next week. PRs are welcome too, if anyone needs it quicker than that.

**Comment by gvisca** [comment]
It would be wonderful if you could have a look on this.
With nuxt 1.1.0, it throws an error
```
Error: nuxt.plugin('renderer',..) is not supported. Use new hooks system.
```

**Comment by DiederikvandenB** [comment]
Fixed with a8168cceb6e9ef0c9653bf7483ae019bad0087e8. Thanks for the PR @Darkside73!
